src/ImageRunModal.jsx

```diff
@@ -997,7 +997,7 @@ export class ImageRunModal extends React.Component {
                                   <Popover aria-label={_("Health check timeout help")}
                                       enableFlip
                                       bodyContent={_("The maximum time allowed to complete the health check before an interval is considered failed.")}>
-                                      <button onClick={e => e.preventDefault()} className="pf-c-form__group-label-help">
+                                      <button onClick={e => e.preventDefault()} className="pf-v5-c-form__group-label-help">
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1299-20230606-125816-eb1c0f76-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)